### ISSUE TITLE
Exploiting Dimensions with low power

### ISSUE THREAD

**Issue by Euphoreich** [McJty/RFTools#490]
Love the mod, love the feature where everything stops ticking when powers at 0. Right now people are able to exploit this by running a very low power line to the dimension builder keeping tick time going, without having to send in all the rf needed. 

Suggested change would be to not start tick time up again until its been powered continuously for some length of time. (maybe 20 seconds or so.)

**Comment by McJty** [comment]
yes, this is something that I have been considering and will probably add at some point
